src/tests/ftest/datamover/copy_basics.py

```diff
@@ -0,0 +1,354 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+'''
+from data_mover_test_base import DataMoverTestBase
+from os.path import join, sep
+from apricot import skipForTicket
+
+class CopyBasicsTest(DataMoverTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test class for basic Datamover validation
+
+    Test Class Description:
+        Tests basic functionality of the datamover utility.
+        Tests the following cases:
+            Copying between UUIDs, UNS paths, and external POSIX systems.
+            Copying between pools.
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a CopyBasicsTest object."""
+        super(CopyBasicsTest, self).__init__(*args, **kwargs)
+
+    def setUp(self):
+        """Set up each test case."""
+        # Start the servers and agents
+        super(CopyBasicsTest, self).setUp()
+
+        # Get the parameters
+        self.flags_write = self.params.get(
+            "flags_write", "/run/ior/copy_basics/*")
+        self.flags_read = self.params.get(
+            "flags_read", "/run/ior/copy_basics/*")
+        self.test_file = self.params.get(
+            "test_file", "/run/ior/copy_basics/*")
```

Comment:
I used `self.test_file` in this case because that value is added to many different paths. I can't rely on `self.ior_cmd.test_file.value`, because ior is ran many times for different paths, such as `/testFile`, `/dir1/testFile`, `/dir2/testFile`


> And use self.ior_flags[0] for write and self.ior_flags[1] for read.

I noticed some other tests did it this way, but I decided not to because I felt it wasn't as self-documenting. But I can update this to be consistent, which is probably more important.

Reply:
Hmm. I think I could actually do something like this for my purposes:
```
def setUp(self):
    ...
    self.test_file = self.ior_cmd.test_file.value
    # Now I have the "original" value that I can use to append to paths later on
```